### ISSUE TITLE
Lightpack v6 board led disable option

### ISSUE THREAD

**Issue by HardLight** [woodenshark/Lightpack#63]
Could lightpack board led be disabled or be on only then data is sent to board?
It's annoying then light is still on system standby.

**Comment by aureq** [comment]
Maybe look at https://github.com/woodenshark/Lightpack/pull/66

**Comment by HardLight** [comment]
@aureq solution worked fine, thanks!

**Comment by aureq** [comment]
@HardLight Did you managed to compile it and is it working for you? Do you know with HW version you have?

I compiled on Debian 8 and flashed the resulting HEX file but doesn't seem to work for me. I use the (old) kickstarter hardware.

It would be nice to get an official firmware release.

**Comment by HardLight** [comment]
@aureq Yes it's working for me on hw v6. Make sure you uncomment  ENCLOSURE_LED_OFF flag in LUFAConfig.h

**Comment by aureq** [comment]
@HardLight Yes, exactly what I did too and I compiled the code in Debian 8.2. However, the device isn't usable afterward and I have no way to tell why.
What's your compilation environment by any chance ? And how have you identified the exact HW version you have ?
Thanks

**Comment by HardLight** [comment]
@aureq Environment shouldn't be a concern if it builds without errors. My lightpack is from DIY schemes provided, so its v6. IDK about kickstarter ver but try flashing all firmware hex from v4 to v7 and check which works.
